nodes/src/nodes/chart_chartjs/IInstance.py

```
@@ -24,55 +24,134 @@
+        ),
+    )
+    def generate_chart(self, args):
+        """
+        Generate a Chart.js v4 chart configuration from data via the pipeline LLM.
+        """
+        if not isinstance(args, dict):
+            raise ValueError('Tool input must be a JSON object')
+
+        data = args.get('data')
+        if data is None:
+            raise ValueError('"data" is required')
```

Ruff (0.15.7)

[warning] Avoid specifying long messages outside the exception class

(TRY003)